apps/studio/hooks/analytics/useLogsQuery.tsx

```diff
@@ -27,6 +31,10 @@ const useLogsQuery = (
   projectRef: string,
   initialParams: Partial<LogsEndpointParams> = {}
 ): LogsQueryHook => {
+  // [Joshen] TODO once API is unblocked
```

Comment:
Sorry what are we TODOing? 😅

Reply:
sorry! should be more clear in my comments here 😬  i'll update the comment so its clearer!
but all the TODOs here are for the API reports page which BE support is not ready just yet